tests/fixtures/filter_ntfs/output_extract/trips.txt

```diff
@@ -1,3 +1,3 @@
 trip_id,route_id,physical_mode_id,dataset_id,service_id,trip_headsign,trip_short_name,block_id,company_id,trip_property_id,geometry_id
```

Comment:
I still don't understand why you do reorder all the file's lines since the test utility is checking for a `Set` of line anyway (so the order doesn't matter).

Reply:
I don't reorder anything, the code do it, but I don't where. in sanitize a lot of things happen

Reply:
ok I understand what you are saying. I'll change the fixtures

Reply:
I just executed the binary to see if there are changes

Reply:
fixed now